deluge/tests/test_ui_entry.py

```diff
@@ -391,19 +392,22 @@ async def test_console_command_status(self):
         self.patch_arg_command(['status'])
         self.patch(sys, 'stdout', fd)
 
-        await self.exec_command()
+        with mock.patch('deluge.ui.console.main.ConsoleUI.quit', autospec=True):
+            await self.exec_command()
 
         std_output = fd.out.getvalue()
         assert std_output.startswith('Total upload: ')
         assert std_output.endswith(' Moving: 0\n')
 
-    @defer.inlineCallbacks
+    @pytest_twisted.inlineCallbacks
     def test_console_command_config_set_download_location(self):
         fd = StringFileDescriptor(sys.stdout)
         self.patch_arg_command(['config --set download_location /downloads'])
         self.patch(sys, 'stdout', fd)
 
-        yield self.exec_command()
+        with mock.patch('deluge.ui.console.main.ConsoleUI.quit', autospec=True):
```

Comment:
This was calling the real `ConsoleUI.quit()`, which was stopping the reactor and thus making the tests bo boom.